website/src/content/blog/2025-03-31-typespec-1-0-release/type_spec_1_0_release.md

```diff
@@ -0,0 +1,188 @@
+---
+slug: 2025-03-31-typespec-1-0-release
+title: "Introducing TypeSpec 1.0-RC: APIs at the speed of thought"
+image: ./TypeSpec_1_0_RC.png
+description: "TypeSpec 1.0-RC is here! Accelerate your API development by automatically generating server-side code, client libraries, schemas, and documentation directly from concise API models, enabling you to focus on innovation rather than implementation details."
+publishDate: 2025-03-19
+author:
+  name: Mario Guerra
+  title: Senior Product Manager @ Microsoft
+authorAvatar: assets/img/authors/mario_guerra.png
+tags:
+  - release
+  - announcement
+---
+
+Ever had a great idea for a new service, only to get bogged down by tedious boilerplate code and manual API specifications? With TypeSpec, you can now turn your API ideas into reality almost as quickly as you think of them.
+
+Okay, so maybe not _that_ quickly, but you get the idea. TypeSpec has been in <a href="/blog/2024-04-25-introducing/" target="_blank" rel="noopener noreferrer">development for while now</a>, and we're excited to be marking a significant milestone for the TypeSpec project with the unveiling of the 1.0 Release Candidate. The 1.0-RC release is a culmination of our efforts to create a powerful, flexible, and user-friendly API design language that meets the needs of developers and organizations alike, both inside and outside of Microsoft.
```

Comment:
I'm wondering if we should refer to this as the "TypeSpec Language release candidate" just to be clear that the entire product that is typespec is not RC? I don't know. Still worried about causing confusion about what we're doing here.